packages/ui/src/Toolbar/index.js

```diff
@@ -40,7 +40,8 @@ import Provider from '../Provider'
 
 type Props = {
   isInsertMode: boolean,
-  editor: Editor
+  editor: Editor,
+  pluginsNotFoundText: any
```

Comment:
This should be `string`, right?

Reply:
Not rly, normally i would do string | JSX.Element but I don't know how that works in flow. The thing is some folks might want to render some real jsx content. take "<h1>Test</h1>" as an example. Locking it down to string would restrict that

Reply:
Ah, I see, the right value would then be `Node`:

```
import type {Node} from 'react'

//...

  pluginsNotFoundText: Node

```

In that case I also want to propose to rename the property from `pluginsNotFoundText` to `noPluginFoundContent` or something similar - it should be clear that it can be text, an image, a react element, or whatever.

Reply:
Makes sense, fixed in latest commit